### PR TITLE
Added a static version of the wxWidgets crossbuild

### PR DESCRIPTION
**Pull request by larspalo**
With this PR a static version of the wxWidgets library is also crosscompiled and available for usage.

### CODE REVIEW

File: build-scripts/for-win64/build-static-on-ubuntu.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+cd ..
+
+mkdir $PKG_DIR/DEBIAN
+
+cat >$PKG_DIR/DEBIAN/control <<EOF
+Package: $DEBIAN_PKG_NAME
+Version: $VERSION-$BUILD_VERSION
+Architecture: all
+Maintainer: Oleg Samarin <osamarin68@gmail.com>
```
Comment (oleg68) [inline build-scripts/for-win64/build-static-on-ubuntu.sh:37]: Do you want to become an owner of wxwidgets3-mingw-w64-static?

File: build-scripts/for-win64/build-static-on-ubuntu.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+SRC_DIR=`dirname $0`/../..
+PARALLEL_PRMS="-j$(nproc)"
+DEBIAN_PKG_NAME=wxwidgets3-mingw-w64-static
+VERSION=${1:-3.2.3}
+BUILD_VERSION=${2:-0.go}
+MINGW64_PREFIX=/usr/x86_64-w64-mingw32
+PKG_DIR=`pwd`/build/win64/${DEBIAN_PKG_NAME}_${VERSION}-${BUILD_VERSION}_all
+
+
+mkdir -p build/win64
```
Comment (oleg68) [inline build-scripts/for-win64/build-static-on-ubuntu.sh:14]:
```suggestion
mkdir -p build/win64-static
```

And everyewhere later.

Reason: if someone debugs both build types in a local environment, they should not conflict.

**Review by oleg68 — CHANGES_REQUESTED** [review]

File: build-scripts/for-win64/build-static-on-ubuntu.sh (hunk 1 above)
Comment (larspalo) [reply build-scripts/for-win64/build-static-on-ubuntu.sh:37]: Sure I can change it if it's ok with you.

File: build-scripts/for-win64/build-static-on-ubuntu.sh (hunk 2 above)
Comment (larspalo) [reply build-scripts/for-win64/build-static-on-ubuntu.sh:14]: Ok, I'll change that.

Comment (larspalo) [reply build-scripts/for-win64/build-static-on-ubuntu.sh]: @oleg68 In addition to your suggestions above, I also added some changes to get rid of the deprecated warnings when running the workflow file. It's the same type of changes that are already applied to the GO workflow file.

**Comment by oleg68** [comment]
@larspalo Do you have sufficient permissions for merging this PR by youself? Or I need to change the access rights?

**Pull request merged**